src/CommandLine/Endpoint.cs

```diff
@@ -35,6 +35,15 @@ public static async Task Create(ManagementClient client, CommandArgument name, C
             {
                 Console.WriteLine("Subscription already exists, skipping creation");
             }
+
+            try
+            {
+                await Rule.Create(client, name, topicName, subscriptionName, subscribeTo);
+            }
+            catch (MessagingEntityAlreadyExistsException)
+            {
+                Console.WriteLine("Rule already exists, skipping creation");
```

Comment:
There might be an issue here, a rule might exist with the same name, but the SQL filter would be different.
As a precaution, I would suggest modifying the message here to indicate a check is required.